docs/content/reference/SLOs.md

```diff
@@ -0,0 +1,23 @@
+# SLOs
+This project is committed to satisfy a number of internal SLOs. 
+These SLOs can be taken as reference by consumers to aggregate them and help define their own SLOs.
+
+These SLOs/SLIs are currently just referential and monitored as part of our CI runs.
+They are valid under the following circumstances:
+
+- Running raw Hypershift in a regular OCP management cluster. I.e. Just Hypershift operator and bare HostedCluster and NodePool resources.
+- Management cluster has 2 m6i.4xlarge instances with 16 cores to allocate HostedClusters.
+- No more than 20 concurrent HostedClusters.
+
+## SLOs/SLIs
+- Cluster time to availability should be < 5min.
+- Cluster complete rollout take < 15min.
+- Cluster Memory consumption per HostedCluster should be < 7GB.
+- NodePool time to availability should be < 8min.
```

Comment:
Maybe in the future we could probably get more fine grained here and specify:
- time to instance creation 
- time to node joining cluster
- time for node to become active after joining